src/Conda.jl

```diff
@@ -137,34 +191,40 @@ function _install_conda(force=false)
             end
         end
         Conda.add_channel("defaults")
+        # Update conda because conda 4.0 is needed and miniconda download installs only 3.9
+        run(_set_conda_env(`$(Conda.conda) update -y conda`))
+    end
+    if !isdir(prefix(env))
+        # conda doesn't allow totally empty environments. using zlib as the default package
+        run(_set_conda_env(`$(conda_bin(RootEnv)) create -y -p $(prefix(env)) zlib`, RootEnv))
     end
 end
 
 "Install a new package."
-function add(pkg::AbstractString)
-    _install_conda()
-    run(_set_conda_env(`$conda install -y $pkg`))
+function add(pkg::AbstractString, env::Environment=RootEnv)
+    _install_conda(env)
+    run(_set_conda_env(`$(conda_bin(env)) install -y $pkg`, env))
```

Comment:
It seems like we should just have a function
```jl
runconda(args::Cmd, env::Environment=RootEnv) =
    run(_set_conda_env(`$(conda_bin(env)) $args`, env))
```
and then we can just do
```
runconda(`install -y $pkg`, env)
```
etc. rather than re-typing `run(_set_conda_env($(conda_bin(env))  ...` several times